packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -394,7 +347,7 @@ export default class Tile extends Surface {
       }
       return;
     }
-
+    // --------------------------------------//
```

Comment:
not sure what this bar is adding or signifying?

Reply:
left over comment from when I was troubleshooting something. Will remove